test/tests/docker-build.sh

```diff
@@ -30,4 +30,4 @@ fi
 
 cp -RL "$dir" "$tmp/dir"
 
-command docker build -t "$imageTag" "$tmp" > /dev/null
+error=$(command docker build -t "$imageTag" "$tmp" >&0) || { echo "$error" >&2; exit 1; }
```

Comment:
I think the `0` here is a typo. Probably don't want to try to redirect stdout to stdin.

Reply:
No, it wouldn't work without the 0 there, just having the `>` caused a syntax error